.gitignore

```diff
@@ -6,3 +6,4 @@
 /tests/acceptance.conf.php
 /tests/smoke.conf.php
 /vendor/
+/.idea/
```

Comment:
It should indeed be removed as this is something that needs to be configured locally instead.